File: Data Structures & Algorithms/subarray-sum-equals-k/submission-0.cpp
class Solution {
public:
    int subarraySum(vector<int>& nums, int k) {
 int sum=0;
 int count=0;
 int n=nums.size();
        for(int i=0;i<n;i++){
            for(int j=i;j<n;j++){
            sum+=nums[j];
            if(sum==k) count++;
                 
            }
            sum=0;
        }
        return count;
        
    }
};

File: Data Structures & Algorithms/special-array-with-x-elements-greater-than-or-equal-x/submission-0.java
class Solution {
    public int specialArray(int[] nums) {

      int size=nums.length;

      if(size==0){
        return 0;
      }
        for(int i=1;i<=size;i++){
           int count=0;
           for(int j=0;j<size;j++){
              
              if(nums[j]>=i){
                count++;
              }


           }

           if(count==i){
            return count;
           }
        }
        return -1;
    
    }
}